tests/arp/test_wr_arp.py

```diff
@@ -0,0 +1,175 @@
+import json
+import logging
+import pytest
+
+from common.platform.ssh_utils import prepare_testbed_ssh_keys as prepareTestbedSshKeys
+from ptf_runner import ptf_runner
+
+logger = logging.getLogger(__name__)
+
+# Globals
+PTFRUNNER_QLEN = 1000
+VXLAN_CONFIG_FILE = '/tmp/vxlan_decap.json'
+
+class TestWrArp:
+    '''
+    TestWrArp Performs control plane assisted warm-reboo
+    '''
+    def __prepareVxlanConfigData(self, duthost, ptfhost):
+        '''
+        Prepares Vxlan Configuration data for Ferret service running on PTF host
+
+        @param duthost: AnsibleHost object of DUT
+        @param ptfhost: AnsibleHost object of PTF
```

Comment:
Let's move to the new comment style?

Reply:
Actually, I did. I am not sure if your browser it viewing stale PR or pointing to an earlier commit.

Reply:
Yes, I see from the log here that the file was outdated!